src/custom/simplifions/components/SimplifionsSolutionDescription.vue

```diff
@@ -64,23 +60,23 @@
             Sommaire
           </h2>
           <ol>
-            <li>
+            <li v-if="solution.Description_longue">
               <a
                 id="summary-link-1"
                 href="#possibilites-simplification"
                 class="fr-summary__link"
                 >Possibilités de simplification</a
               >
             </li>
-            <li>
+            <li v-if="relatedCasUsages.length > 0">
```

Comment:
Nitpick: do you need that `> 0`? cf https://github.com/opendatateam/udata-front-kit/pull/824/files#diff-c30dd03fc1200f1c468229a7f72a32ddb1bcece7588175f853fbf6dfa564c3adR35

Reply:
Probably not, will fix